README.md

```diff
@@ -130,6 +130,12 @@ To install the Python shared library on Ubuntu:
 sudo apt install python3-dev
 ```
 
+To install the Python shared library on Fedora:
```

Comment:
I think it can be generalized to "RPM-based distributions"?

Reply:
I think this is complicated. SUSE uses `zypper`, the still supported RHEL 7 uses `yum`. The package name seems to be pretty universal though.

Reply:
Yeah, I meant that it should then say "install the package `python3-devel`".

Reply:
OK, so let's say
```suggestion
To install the Python shared library on RPM based distributions (e.g. Fedora, Red Hat, SuSE), install the `python3-devel` package.
```
and remove the `dnf` snippet below.